scripts/build.sh

```diff
@@ -29,13 +29,6 @@
 
 export PATH="/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
 
-delete_tmp_manifest(){
```

Comment:
I think you need to check your merge history - this section should not be getting removed.

Reply:
Returned